src/Honeybee_orientHBGlz.py

```diff
@@ -0,0 +1,71 @@
+#
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
+# 
+# This file is part of Honeybee.
+# 
+# Copyright (c) 2013-2016, Mostapha Sadeghipour Roudsari <Sadeghipour@gmail.com> 
+# Honeybee is free software; you can redistribute it and/or modify 
+# it under the terms of the GNU General Public License as published 
+# by the Free Software Foundation; either version 3 of the License, 
+# or (at your option) any later version. 
+# 
+# Honeybee is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+
+"""
+Use this component to set the orientation(s) of the glazing.
+
+-
+Provided by Honeybee 0.0.59
+
+    Args:
+        _orientation: The orientation/s that you'd like to add glazing to.
+        _ratio: The window to wall ratio.
+    Returns:
+        _glzratio: A list of glazing orientations that can be plugged into (glazingCreator).
+"""
+
+import rhinoscriptsyntax as rs
+import Rhino as rc
+import scriptcontext as sc
+import os
+import sys
+import System
+import Grasshopper.Kernel as gh
+import uuid
+
+ghenv.Component.Name = 'Honeybee_orientHBGlz'
+ghenv.Component.NickName = 'orientHBGlz'
+ghenv.Component.Message = 'VER 0.0.59\nJAN_26_2016'
+ghenv.Component.IconDisplayMode = ghenv.Component.IconDisplayMode.application
+ghenv.Component.Category = "Honeybee"
+ghenv.Component.SubCategory = "00 | Honeybee"
+#compatibleHBVersion = VER 0.0.57\nNOV_01_2015
+#compatibleLBVersion = VER 0.0.59\nFEB_01_2015
+try: ghenv.Component.AdditionalHelpFromDocStrings = "3"
+except: pass
+
+o_list = ['North', 'East', 'South', 'West']
+_glzRatio = [0,0,0,0]
+
+
+for a in xrange(0,len(_orientation)):
+    
+    i = o_list.index(_orientation[a])
+
+    _glzRatio[i] = _ratio
+    
+
```

Comment:
Also in case you want your code to work for "SOUTH" or "south" as well as "South" I would suggest to compare ```lower()``` case of both cases together.
```python
o_list = ['north', 'east', 'south', 'west']
for a in _orientation:
   i = o_list.index(a.lower())
   # ...
```

Finally to make sure your code won't fail if someone inputs soth for example, you need to catch the exception for index and give an appropriate error message.
```python
o_list = ['north', 'east', 'south', 'west']
for a in _orientation:
  try:   
    i = o_list.index(a.lower())
  except ValueError:
    print "%s is not a valid orientation." % str(a)
  finally:
    # input is valid, let's assign the direction
    _glzRatio[i] = _ratio
```